packages/graphql/src/schemaTypes/objectTypes/gql-Mutation.ts

```diff
@@ -593,5 +593,20 @@ export const mutation = mutationType({
         return true
       },
     })
+
+    t.field('setTestingTypeAndReconfigureProject', {
+      description: 'Set the selected testing type, and reconfigure the project',
+      type: Query,
+      args: {
+        testingType: nonNull(arg({ type: TestingTypeEnum })),
+      },
+      resolve: async (source, args, ctx) => {
+        ctx.actions.project.setForceReconfigureProjectByTestingType(args.testingType)
+        ctx.actions.project.setCurrentTestingType(args.testingType)
```

Comment:
Why does do you need to re-set the current testing type if it's already set?

Reply:
In this mutation the testing type is not set yet - b/c it's the mutation that happens when you select reconfigure from the testing type card - so, it sets the flag, and then sets the testing type to move forward on the setup